app/src/main/java/com/google/android/samples/socialite/ui/home/Home.kt

```diff
@@ -48,11 +56,27 @@ import androidx.navigation.compose.rememberNavController
 import com.google.android.samples.socialite.R
 import com.google.android.samples.socialite.ui.AnimationConstants
 import com.google.android.samples.socialite.ui.home.timeline.Timeline
+import com.google.android.samples.socialite.ui.player.findActivity
 
+@OptIn(ExperimentalMaterial3WindowSizeClassApi::class)
 @Composable
 fun Home(
     onChatClicked: (chatId: Long) -> Unit,
     modifier: Modifier = Modifier,
+) {
+    val windowSizeClass = calculateWindowSizeClass(
+        activity = LocalContext.current.findActivity()
+    ).widthSizeClass
+    when (windowSizeClass) {
+        WindowWidthSizeClass.Compact -> CompactScreen(modifier, onChatClicked)
+        else -> LargeScreen(onChatClicked = onChatClicked)
```

Comment:
Because `CompactScreen` and `LargeScreen` are being called in two different places, the internal state within each (including the `NavHost`) will be lost when the window size changing results in flipping between whether the `CompactScreen` and `LargeScreen` is being used.

Could we use `NavigationSuiteScaffold` from `material3-adaptive` instead of needing to do this manually?